.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       - node/with-cache:
           steps:
             - run: yarn install
-            - run: yarn run fix
```

Comment:
why did this get deleted

Reply:
we replaced fix by test because yarn fix did not work

Reply:
because yarn fix did not work so we replace it by test

Reply:
this file should have no changes (revert the changes) it is correct as it is